Doc/tools/clean-files.txt

```diff
@@ -0,0 +1,6 @@
+
+Doc/library/gc.rst
+
+Doc/library/sqlite3.rst
+
+Doc/whatsnew/3.12.rst
```

Comment:
I don't want to add too much feature creep in the PR, but it would be useful to add a brief comment at the top to say:
1. what the file is for and who uses it;
2. why there are blank lines.

The code can then be updated to something like this to ignore the comments:
```py
with Path("Doc/tools/clean-files.txt").open() as clean_files:
    CLEAN = [fname for fname in clean_files
             if fname.strip() and not fname.startswith('#')]
```

Reply:
Yup, I was thinking of requesting the same, but I felt I'd already asked for enough, heh

Reply:
Done!